Summary Table/Summary Table Markdown Development.Rmd

```diff
@@ -0,0 +1,123 @@
+---
+title: "&#32;"
+header-includes:
+  - \usepackage{colortbl}
+output:
+  html_document:
+---
+
+```{r setup, include=FALSE}
+
+```
+
+
+
+
+## Summary Table
+<!--# below to convert to a separate script and call-->
+<!--# Clive to create dummy vars with same names as called vars to populate table -->
```

Comment:
Are these comments still relevant (@CliveWG)

Reply:
Thanks - no they can be deleted, they were used during the initial stages of creating the table.